src/core/control/tools/EditSelection.cpp

```diff
@@ -9,8 +9,6 @@
 #include <string>     // for string
 #include <utility>
 
-#include <gdk/gdk.h>  // for gdk_cairo_set_sour...
-
```

Comment:
I guess this include removal should have gone into `Selector.cpp`. We still have `gdk_cairo_set_source_rgba` in `EditSelection.cpp`.